tests/jsx-multiline-assign/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -22,22 +22,12 @@ const comp4 = <div style={styles} key=\\"something\\">Create wrapping parens and
 const comp5 = <div>Keep it on one line.</div>;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 const comp1 = (
-  <div style={styles} key=\\"something\\">
-    Keep the wrapping parens.
-  </div>
+  <div style={styles} key=\\"something\\">Keep the wrapping parens.</div>
 );
 
-const comp2 = (
-  <div style={styles} key=\\"something\\">
-    Create wrapping parens.
-  </div>
-);
+const comp2 = <div style={styles} key=\\"something\\">Create wrapping parens.</div>;
```

Comment:
we'd need to change these tests, as they no longer check the intended behavior (can just make text longer). Probably true for several others here.